src/EditorFeatures/CSharpTest/ChangeSignature/AddParameterTests.cs

```diff
@@ -0,0 +1,1148 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ChangeSignature;
+using Microsoft.CodeAnalysis.Editor.UnitTests.ChangeSignature;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities.ChangeSignature;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ChangeSignature
+{
+    public partial class ChangeSignatureTests : AbstractChangeSignatureTests
+    {
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task AddParameters()
+        {
+            var markup = @"
+static class Ext
+{
+    /// <summary>
+    /// This is a summary of <see cref=""M(object, int, string, bool, int, string, int[])""/>
+    /// </summary>
+    /// <param name=""o""></param>
+    /// <param name=""a""></param>
+    /// <param name=""b""></param>
+    /// <param name=""c""></param>
+    /// <param name=""x""></param>
+    /// <param name=""y""></param>
+    /// <param name=""p""></param>
+    static void $$M(this object o, int a, string b, bool c, int x = 0, string y = ""Zero"", params int[] p)
+    {
+        object t = new object();
+
+        M(t, 1, ""two"", true, 3, ""four"", new[] { 5, 6 });
+        M(t, 1, ""two"", true, 3, ""four"", 5, 6);
+        t.M(1, ""two"", true, 3, ""four"", new[] { 5, 6 });
+        t.M(1, ""two"", true, 3, ""four"", 5, 6);
+
+        M(t, 1, ""two"", true, 3, ""four"");
+        M(t, 1, ""two"", true, 3);
+        M(t, 1, ""two"", true);
+
+        M(t, 1, ""two"", c: true);
+        M(t, 1, ""two"", true, 3, y: ""four"");
+
+        M(t, 1, ""two"", true, 3, p: new[] { 5 });
+        M(t, 1, ""two"", true, p: new[] { 5 });
+        M(t, 1, ""two"", true, y: ""four"");
+        M(t, 1, ""two"", true, x: 3);
+
+        M(t, 1, ""two"", true, y: ""four"", x: 3);
+        M(t, 1, y: ""four"", x: 3, b: ""two"", c: true);
+        M(t, y: ""four"", x: 3, c: true, b: ""two"", a: 1);
+        M(t, p: new[] { 5 }, y: ""four"", x: 3, c: true, b: ""two"", a: 1);
+        M(p: new[] { 5 }, y: ""four"", x: 3, c: true, b: ""two"", a: 1, o: t);
+    }
+}";
+            var updatedSignature = new[] {
+                new AddedParameterOrExistingIndex(0),
+                new AddedParameterOrExistingIndex(2),
+                new AddedParameterOrExistingIndex(new AddedParameter(null, "int", "newIntegerParameter", "12345"), "System.Int32"),
```

Comment:
other tests had `int` and `int`.  why is this `int` and `System.Int32`?

Reply:
Filed https://github.com/dotnet/roslyn/issues/42734 to clean this all up.